src/olympia/lib/tests/test_session.py

```diff
@@ -0,0 +1,13 @@
+from settings import SESSION_COOKIE_AGE, SESSION_EXPIRE_AT_BROWSER_CLOSE
```

Comment:
I'm actually not sure if this is a good way. There are multiple times imho where we don't have a `settings` module in `$PYTHONPATH` so I'd change this import to `from django.conf import settings` and use these values from there.

You can't catch all overwrites anyway.